src/jsone.erl

```diff
@@ -288,6 +292,13 @@
 %% - If the value is `last' then the last duplicate key/value is returned.
 %% - default: `first'<br />
 %%
+%% `stream': <br />
+%%
+%% Decode the input in multiple chunks. Instead of a result or error,
+%% `{incomplete, fun()}' is returned. The returned fun takes a single argument
+%% and it should called to continue the decoding. When all the input has been
+%% provided, the fun should be called with `end_stream' or `end_json' to signal
```

Comment:
IMO, it's sufficient to provide either of `end_stream` or `end_json` if those have the same meaning.
What do you think of that?

Reply:
I agree. I added both because jsx accepts both. We can use `end_stream` only.

Reply:
👍  > We can use end_stream only.